ui.iss

```diff
@@ -346,19 +355,19 @@ begin
   lblNewSaveFolder := TLabel.Create(PageSinglePlayer);
   lblNewSaveFolder.Parent := PageSinglePlayer.Surface;
   lblNewSaveFolder.Caption := 'Store save game files in a different folder';
-  lblNewSaveFolder.Top := ScaleY(160);
+  lblNewSaveFolder.Top := ScaleY(170);
   lblNewSaveFolder.Left := ScaleX(20);
   
   descNewSaveFolder := TNewStaticText.Create(PageSinglePlayer);
   descNewSaveFolder.Parent := PageSinglePlayer.Surface;
   descNewSaveFolder.WordWrap := True;
-  descNewSaveFolder.Top := ScaleY(180);
+  descNewSaveFolder.Top := ScaleY(190);
```

Comment:
I learnt a better way of doing these. You can do

`descNewSaveFolder.Top := lblNewSaveFolder.Top + 20`

Means you wont have to adjust everything manually each time you want to reposition

Reply:
Might be +ScaleY(20) might need to experiment